src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitToggleButton.razor.cs

```diff
@@ -31,35 +42,49 @@ public bool IsChecked
             }
         }
 
+        /// <summary>
+        /// Callback that is called when the IsChecked parameter changed.
+        /// </summary>
         [Parameter] public EventCallback<bool> IsCheckedChanged { get; set; }
 
         /// <summary>
-        /// the icon that shows in the button
+        /// The icon that shows in the button.
         /// </summary>
         [Parameter] public string? IconName { get; set; }
 
         /// <summary>
-        /// the text that shows in the label
+        /// The text that shows in the label.
         /// </summary>
         [Parameter] public string? Label { get; set; }
 
         /// <summary>
-        /// URL the link points to, if provided, button renders as an anchor
+        /// URL the link points to, if provided, button renders as an anchor.
         /// </summary>
         [Parameter] public string? Href { get; set; }
 
         /// <summary>
-        /// If Href provided, specifies how to open the link
+        /// If Href provided, specifies how to open the link.
         /// </summary>
         [Parameter] public string? Target { get; set; }
 
         /// <summary>
-        /// The title to show when the mouse is placed on the button
+        /// The title to show when the mouse is placed on the button.
         /// </summary>
         [Parameter] public string? Title { get; set; }
 
+        /// <summary>
+        /// Toggles the state when the button is clicked.
```

Comment:
please change this line to something like "A callback that is called when the button is clicked"